mmcv/ops/point_sample.py

```diff
@@ -153,8 +214,12 @@ def point_sample(input, points, align_corners=False, **kwargs):
     if points.dim() == 3:
         add_dim = True
         points = points.unsqueeze(2)
-    output = F.grid_sample(
-        input, denormalize(points), align_corners=align_corners, **kwargs)
+    if torch.onnx.is_in_onnx_export():
+        output = bilinear_grid_sample(
```

Comment:
Could you explain why we have this difference for exporting and the normal Pytorch pipeline?

Reply:
ONNX does not support pytorch's operation `grid_sample` and during export we should not use it, so I implemented this operation in supported operations that can be exported and inferenced without any issues. [Here](https://github.com/pytorch/pytorch/issues/27212) is related issue in pytorch.

Reply:
There's a `grid_sampler` op for ONNXRuntime as custom op from this PR #916, could you check with that?

Reply:
I tried the custom implementation. Model converted to onnx but I had the following runtime error during loading a model:
```
onnxruntime.capi.onnxruntime_pybind11_state.Fail: [ONNXRuntimeError] : 1 : FAIL : Load model from mmcv_grid_sample.onnx failed:Fatal error: grid_sampler is not a registered function/op
```
My implementation converted and works in onnx runtime. [Here](https://github.com/DmitriySidnev/mmcv/blob/ds/grid_sample_test/grid_sample_test.py) is the script used for test. B.t.w. I did not manage to install `mmcv-full` module with onnx runtime ops via pip. Only from sources.

Reply:
Did you install mmcv-full by following the instruction from [the doc](https://github.com/open-mmlab/mmcv/blob/master/docs/onnxruntime_op.md#how-to-build-custom-operators-for-onnx-runtime)?

Reply:
Yes, I used the instruction. I meant that command `pip install mmcv-full` with environment variable `MMCV_WITH_ORT=1` does not work without `onnxruntime` library that should be configured before.

Reply:
You could try to install mmcv-full from source following the instruction from [the doc](https://github.com/open-mmlab/mmcv/blob/master/docs/onnxruntime_op.md#how-to-build-custom-operators-for-onnx-runtime). `grid_sampler` should work for onnxruntime backend.

Reply:
Yes, it works and seems that my implementation is slower in onnx runtime. But I propose to use my implementation during exporting when custom ops not installed.

Reply:
It could be added. You could add the unit tests for your implementation for both PyTorch and pytorch2onnx.

Reply:
Do you mean tests for `bilinear_grid_sample` function?

Reply:
Yes.

Reply:
Tests added.